lib/edda-server/src/change_set_processor_task/materialized_view.rs

```diff
@@ -530,6 +589,12 @@ async fn build_mv_inner(
                     if let Some(frontend_object) = maybe_frontend_object {
                         frontend_objects.push(frontend_object);
                     }
+                    build_count += 1;
+                    if build_duration > build_max_elapsed {
+                        build_max_elapsed = build_duration;
+                        build_slowest_mv_kind = kind.to_string();
```

Comment:
I believe that if we added strum's [`IntoStaticStr`](https://docs.rs/strum/latest/strum/derive.IntoStaticStr.html) on `ReferenceKind`, then we'd have a `&'static str` representation of kind and it would save us the String allocation here.